doc/Makefile

```diff
@@ -45,13 +48,18 @@ clean:
 	rm -rf generated/*
 	rm -rf modules/*
 
+sync_git:
+	git clone git@github.com:benchopt/benchmark_logreg_l2.git $(BENCH_LOGREG_PATH) || (cd $(BENCH_LOGREG_PATH) ; git pull && cd ../../doc)
+	git clone git@github.com:benchopt/benchmark_lasso.git $(BENCH_LASSO_PATH) || (cd $(BENCH_LASSO_PATH) ; git pull && cd ../../doc)
```

Comment:
most of times (if the folder exists) it wil lprint an error message before moving to the pull command. Is it an issue ?

Reply:
fixed